app/templates/components/banner.html

```diff
@@ -1,10 +1,10 @@
 {%- from "components/alerts_icon.html" import alerts_icon -%}
 
-{% macro banner(number_of_alerts, last_updated) %}
+{% macro banner(number_of_alerts, last_updated, lang='en') %}
   <div class="alerts-notification-banner alerts-icon__container alerts-icon__container--48" role="region" aria-labelledby="alerts-notification-banner__title">
     {{ alerts_icon(height=48) }}
     <h2 class="alerts-notification-banner__title govuk-heading-m govuk-!-margin-bottom-1" id="alerts-notification-banner__title">
-      <a class="govuk-link govuk-link--no-visited-state" href="/alerts/current-alerts">{{ number_of_alerts }} current {% if number_of_alerts == 1 %}alert{% else %}alerts{% endif %}</a>
+      <a class="govuk-link govuk-link--no-visited-state" href="/alerts/current-alerts{% if lang == 'cy' %}.cy{% endif %}">{{ number_of_alerts }} current {% if number_of_alerts == 1 %}alert{% else %}alerts{% endif %}</a>
```

Comment:
Would it be worth creating a create link macro which you can pass a language into which automatically adds the .cy if required?

Reply:
I can do, but if we create that now, that's going to lead to a flurry of other view tests I would imagine